src/wms/src/main/java/org/geoserver/wms/WMSServiceExceptionHandler.java

```diff
@@ -163,13 +163,13 @@ public void handleServiceException(ServiceException exception, Request request)
     
     private boolean isImageExceptionType(String exceptions) {
         return "application/vnd.ogc.se_inimage".equals(exceptions) || "INIMAGE".equals(exceptions)
-                || "BLANK".equals(exceptions);
+                || "BLANK".equals(exceptions) || "application/vnd.ogc.se_blank".equals(exceptions);
```

Comment:
You'll have to specify this type of exception, if you don't want to get an error message displayed in the map composition (for a getMap request).